src/auditlog/backends/logger.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from django.contrib.contenttypes.models import ContentType
+from django.utils.encoding import smart_text
+
+from auditlog import settings
+from auditlog.models import LogEntry
+from auditlog.backends.base import AuditBackend
+from auditlog.middleware import get_thread_data
+
+
+class LoggerBackend(AuditBackend):
+    """Python logger based backend for creation of audit log entries."""
+    message = '{action} on {content_type_name} {object_pk} by actor {actor_name}'
+    logger = 'audit.log'
```

Comment:
what does this do? I am not seeing references to `self.logger` and `audit.log` is set as the default in `settings.AUDITLOG_LOGGER`. Also, what is the `audit.log` default logger name related to? Any reason it needs to be a descendent of `audit`?

Reply:
I think it was legacy - was using during development but then I moved to a package setting.